Directory.Build.targets

```diff
@@ -15,6 +15,12 @@
     <AndroidUseIntermediateDesignerFile>false</AndroidUseIntermediateDesignerFile>
   </PropertyGroup>
 
+  <PropertyGroup Condition="$(TargetFramework.StartsWith('MonoAndroid'))">
```

Comment:
I think you should put this on the previous `<PropertyGroup/>`:
```
Condition="($(TargetFramework.StartsWith('MonoAndroid')) or '$(TargetPlatformIdentifier)' == 'Android') and '$(AndroidApplication)' != 'true'"
```
What you have here would prevent `Resource.designer.cs` from being created in "app" projects, and I think you want to keep those?

Reply:
👍